appinfo/info.xml

```diff
@@ -34,8 +34,8 @@ To whitelist `127.0.0.0/24` and also `192.168.0.0/24`:
 	<screenshot>https://raw.githubusercontent.com/nextcloud/limit_login_to_ip/master/screenshots/not-authorized.png</screenshot>
 	<screenshot>https://raw.githubusercontent.com/nextcloud/limit_login_to_ip/master/screenshots/admin-settings.png</screenshot>
 	<dependencies>
-		<php min-version="7.1" max-version="7.4" />
-		<nextcloud min-version="16" max-version="21" />
+		<php min-version="8.0" max-version="8.2" />
```

Comment:
Please remove and just always support all versions necessary for the mentioned NC versions. The appstore can't handle subsequent matches at the moment.